independent-projects/arc/processor/pom.xml

```diff
@@ -74,6 +74,7 @@
                         <configuration>
                             <signaturesFiles>
                                 <signaturesFile>./banned-signatures.txt</signaturesFile>
+                                <signaturesFile>${maven.multiModuleProjectDirectory}/.forbiddenapis/banned-signatures-common.txt</signaturesFile>
```

Comment:
This is an independent project. We cannot have dependencies on things outside of it.

Reply:
agree.